### PR TITLE
Bug 1209961 - Rename fonts to bidi-proof names.

### PR DESCRIPTION
**Pull request by autra**
This commit removes any reference to end/beginning semantics on directional
icons: left is the beginning and right is the end only for LTR languages. RTL
languages follow the exact opposite semantic. Instead, we keep the left/right in
names for these icons, and let the user of these fonts deals with bidi csss
rules accordingly.

@fabi1cazenave please nitpick the hell out of me.

### CODE REVIEW

**Comment by fabi1cazenave** [comment]
LGTM, I’m just not sure we want to rename `undo`: to me it relates to clockwise / counter-clockwise, not to left / right. Think of spinners & such…

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-icons",
-  "version": "0.9.2",
+  "version": "0.9.3",
```
Comment (fabi1cazenave) [inline package.json]: This would be a **major** change, so I’d suggest using `0.10.0` for the version number.

File: index.html [hunk 2]
```diff
@@ -491,7 +495,9 @@
 
       <a aria-hidden="true" href="#undo-circular" id="undo-circular" data-icon="undo-circular"></a>
 
-      <a aria-hidden="true" href="#undo" id="undo" data-icon="undo"></a>
+      <a aria-hidden="true" href="#undo-left" id="undo-left" data-icon="undo-left"></a>
+
+      <a aria-hidden="true" href="#undo-right" id="undo-right" data-icon="undo-right"></a>
```
Comment (fabi1cazenave) [inline index.html]: as I said in my comment, I don’t think the `undo` should be mirrored (counter-clockwise meaning).

**Comment by fabi1cazenave** [comment]
To apply such a patch, we’ll have to submit a PR to all web-components that depend on it (if any).

Then, on the gaia side, once this patch lands we’ll have to write a patch that updates:
- the gaia-icons version number in /shared/bower.json
- the gaia-icons directory in /shared/elements/
- all building blocks in /shared/styles
- all apps in /apps, /dev_apps and /tv_apps

For the two last items, a good `grep` script should help… :-/

**Comment by fabi1cazenave** [comment]
LGTM, please open a PR upstream.

**Comment by fabi1cazenave** [comment]
(Note: I can’t merge this, and more generally I’m really not sure it’s worth cloning gaia-components repositories on a private Phoxygen repo)

**Comment by autra** [comment]
Yep, I'm closing this (note that this repo is public, and I just made a PR for the discussion).

**Pull request closed without merging**